torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -107,7 +109,9 @@ bool AliasDb::hasWriters(const Value* v) const {
     return numWrites_ != 0;
   }
 
-  if (!elementMap_.count(v) || v->mustBeNone()) {
+  // bc this is important api, check if value is non-aliasing before wildcard
+  // check
+  if (!elementMap_.count(v) || nonAliasingValue(v)) {
```

Comment:
do we still need this change? Below in `makePointerTo` you are a giving aliases to non-aliasing values anyway

Reply:
yes - as the comment says, this is an important api, so we shouldn't return True for a non-aliasing value if there is a wildcard write in the graph